README.md

```diff
@@ -107,66 +114,78 @@ For more information about configuring your Cloud Pak for Data cluster, see http
 
-Add the component, style, and client library to your application:
+1. Add the component, style, and client library to your application:
 
-```
-yarn add @ibm-watson/discovery-react-components @ibm-watson/discovery-styles ibm-watson
-```
+   ```
+   yarn add @ibm-watson/discovery-react-components @ibm-watson/discovery-styles ibm-watson carbon-components carbon-components-react
+   ```
 
-or
+   or
 
-```
-npm i @ibm-watson/discovery-react-components @ibm-watson/discovery-styles ibm-watson
-```
+   ```
+   npm install --save @ibm-watson/discovery-react-components @ibm-watson/discovery-styles ibm-watson carbon-components carbon-components-react
+   ```
 
-Add the `DiscoverySearch` component with corresponding `searchClient` and optionally any components you would like to use to display Discovery Search Results.
+2. Add `node-sass` as a dev dependency
 
-```jsx
-// src/App.js
+   ```
+   yarn add -D node-sass
+   ```
 
-import React from 'react';
-import {
-  DiscoverySearch,
-  SearchInput,
-  SearchResults,
-  SearchFacets,
-  ResultsPagination,
-  DocumentPreview
-} from '@ibm-watson/discovery-react-components';
-import { CloudPackForDataAuthenticator, DiscoveryV2 } from 'ibm-watson';
-
-// optionally import SASS styles
-import '@ibm-watson/discovery-styles/scss/index.scss';
-// or load vanilla CSS
-// import '@ibm-watson/discovery-styles/css/index.css';
-
-// Replace these values
-const username = '<your cluster username>';
-const password = '<your cluster password>';
-const url = '<your cluster url>';
-const serviceUrl = '<your discovery url>';
-const version = '<YYYY-MM-DD discovery version>';
-const projectId = '<your discovery project id>';
+   or
 
-const App = () => {
-  const authenticator = new CloudPakForDataAuthenticator({ username, password, url });
-  const searchClient = new DiscoveryV2({ url: serviceUrl, version, authenticator });
+   ```
+   npm install --save-dev node-sass
+   ```
 
-  return (
-    <DiscoverySearch searchClient={searchClient} projectId={'<your discovery project id>'}>
-      <SearchInput />
-      <SearchResults />
-      <SearchFacets />
-      <ResultsPagination />
-      <DocumentPreview />
-    </DiscoverySearch>
-  );
-};
-```
+3. Add the `DiscoverySearch` component with corresponding `searchClient` and optionally any components you would like to use to display Discovery Search Results.
+
+   ```jsx
+   // src/App.js
+
+   import React from 'react';
+   import {
+     DiscoverySearch,
+     SearchInput,
+     SearchResults,
+     SearchFacets,
+     ResultsPagination,
+     DocumentPreview
+   } from '@ibm-watson/discovery-react-components';
+   import DiscoveryV2 from 'ibm-watson/discovery/v2';
+   import { BearerTokenAuthenticator } from 'ibm-watson/auth';
+   import '@ibm-watson/discovery-styles/scss/index.scss';
+
+   // Replace these values
+   const bearerToken = '{REPLACE_ME}'; // retrieved from CP4D Admin UI under instance details
```

Comment:
Should we just have the server setup script crab the `bearerToken`, then all this info would be available in that env file. Even if they don't plan on using that example app, there could be one step to run that script and have this info available to them (either for use by the example server or to plug into their own code).

Reply:
yes we could simplify to the bearer token - my only worry is that using the bearer token means it can expire whereas the username/password never expires (the SDK will do the bearer token refresh when needed)